test/spec/test-parallel-uploads.js

```diff
@@ -178,6 +197,190 @@ describe('tus', () => {
       expect(testUrlStorage.removeUpload).toHaveBeenCalled()
     })
 
+    it('should split a file into multiple parts based on custom `parallelUploadBoundaries`', async () => {
```

Comment:
I see that you copied the test case from above but I think we can remove some details form this test as they are already covered in the test case above. I am mainly thinking about removing testUrlStorage, retryDelays (plus the 500 response on line 329), the headers option, the metadata option, the onProgress callback. Could you do this?

Reply:
Done!